.eslintrc.json

```diff
@@ -15,5 +15,13 @@
   "plugins": ["react", "@typescript-eslint"],
   "rules": {
     "react/prop-types": "off"
-  }
+  },
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx"],
+      "rules": {
+        "no-undef": "off"
```

Comment:
why was this rule added?

Reply:
I added this rule to avoid getting es-lint errors for JSX import in the `interface/index.ts` file. We can either add it to globals or disable the `no-undef` rule because TypeScript already has already its own checks. 
I can add `globals` instead of this rule if wanted.

Reply:
Which part was throwing the error?

Reply:
I am assuming it's for React. Why are we not importing React explicitly then?

Reply:
An error was for importing JSX.Element in this file. I'll add globals and improve it.

Reply:
Use React.ReactNode instead of JSX.Element, and you can drop the globals :). I kept it `index.ts` only.